pageserver/src/pgdatadir_mapping.rs

```diff
@@ -1481,10 +1481,7 @@ impl<'a> DatadirModification<'a> {
             // * no aux files -> v1/v2/cross-validation
             // * cross-validation->v2
             if AuxFilePolicy::is_valid_migration_path(current_policy, switch_policy) {
-                self.tline.last_aux_file_policy.store(Some(switch_policy));
-                self.tline
-                    .remote_client
-                    .schedule_index_upload_for_aux_file_policy_update(Some(switch_policy))?;
+                self.tline.force_switch_aux_policy(switch_policy)?;
```

Comment:
Should the function just be called `switch_aux_policy`, since we call from here on the happy path where we're not forcing anything?